packages/element-web/default.nix

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  jq,
+  yarn,
+  fixup_yarn_lock,
+  nodejs,
+  jitsi-meet,
+}: let
+  pinData = import ./pin.nix;
+  inherit (pinData.hashes) webSrcHash webYarnHash;
+  noPhoningHome = {
+    disable_guests = true; # disable automatic guest account registration at matrix.org
+  };
+in
+  stdenv.mkDerivation (finalAttrs:
+    builtins.removeAttrs pinData ["hashes"]
+    // {
```

Comment:
Instead of all this you could just say inside the brackets: `inherit (pinData) version`

Reply:
No changes here, hash is used from pin.nix file